envoy_build_config/extensions_build_config.bzl

```diff
@@ -7,6 +7,7 @@ EXTENSIONS = {
     "envoy.filters.http.buffer":                      "//source/extensions/filters/http/buffer:config",
     "envoy.filters.http.dynamic_forward_proxy":       "//source/extensions/filters/http/dynamic_forward_proxy:config",
     "envoy.filters.http.local_error":                 "@envoy_mobile//library/common/extensions/filters/http/local_error:config",
+    "envoy.http.original_ip_detection.xff":           "//source/extensions/http/original_ip_detection/xff:config",
```

Comment:
this is probably not necessary as its listed in envoy's `all_extensions.bzl` 

if it is its in the wrong alpha order